README.md

```diff
@@ -4,240 +4,77 @@ languages:
 - javascript
 products:
 - nodejs
+- express
 - azure-active-directory
 description: "Add authentication to a Node.js web application with the Microsoft Authentication Library for Node.js (MSAL Node)."
-urlFragment: "quickstart-v2-nodejs-webapp-msal"
+urlFragment: "ms-identity-node"
 ---
 
-# Add user authentication to a Node web app with MSAL
+# Express web application built with MSAL Node and Microsoft identity platform
```

Comment:
>Express web application built with MSAL Node and Microsoft identity platform [](http://example.com/codeflow?start=2&length=76)

nit: Node web application using MSAL to sign-in user with the Microsoft Identity Platform and acquire a token for MS Graph

Reply:
done